src/App.jsx

```diff
@@ -14,33 +15,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortedAlph, setSortedAlph] = useState(false);
+  const [sortedByLength, setSortedByLength] = useState(false);
+  const [reverse, setReverse] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const applySortingAndReverse = (array, sortType) => {
+    const updatedGoods = [...array];
```

Comment:
- Move this function outside the component
- array is a bad name, the name should describe variable/argument/property